### DISCUSSION TITLE
Scratch

### DISCUSSION THREAD

**Discussion by Shluffy** [ZI-Youtuber/ZI-Youtuber#1] [Q&A] [answered]
Do you scratch, and if so what is your scratch account?

**Comment by zoraezimran** [comment, marked as answer]
@Shluffy  I do scratch here is my scratch account: https://scratch.mit.edu/users/ISLAMICPRO2020/

**Reply by Shluffy** [reply]
Thanks.

**Reply by Shluffy** [reply]
I suppose if your muslim it is ramadan now.

**Reply by zoraezimran** [reply]
yep
